### PR TITLE
Updating requirements

### PR DESCRIPTION
**Pull request by stephendonner**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 browserid
```
Comment (davehunt) [inline requirements.txt]: I think this should be `bidpom` now, though updating it can be a separate commit, perhaps even a separate pull.

Comment (davehunt) [inline requirements.txt]: On second thoughts, let's do this in a separate patch as it might not just be a package name change - imports will also need updating.

**Comment by davehunt** [comment]
r+

**Comment by rbillings** [comment]
R+
